docs/interfaces.html

```diff
@@ -176,6 +136,13 @@ <h3>Issue</h3>
             <li><a href="https://w3c-ccg.github.io/ld-proofs/#dfn-challenge">challenge </a> </li>
             <li><a href="https://w3c-ccg.github.io/ld-proofs/#dfn-proof-type">proofType </a> </li>
         </ul>
+
+        Beware that while the challenge comes from the verifier and must be signed over.
```

Comment:
```suggestion
        Beware that while the challenge comes from the verifier and must be signed over.
```
Should this be an introductory phrase in the sentence that follows below?  If so, some capitalization and punctuation changes are necessary.

Else, is it meant to be a standalone?  In this case, it needs some rewording to actually be a sentence.

Reply:
hmm.... for some reason I cannot accept this suggestion...